Add content/pt-br/TLS(Transport Layer Security).md (#759)
cncf/glossary: content/pt-br/TLS(Transport Layer Security).md

---
title: TLS
status: Completed

> category: Conceito

Reviewer: ```suggestion
category: conceito
```

---

## O que é isso

> Transport Layer Security (TLS) é um protocolo projetado para fornecer maior segurança à comunicação através de uma rede. Garante a entrega segura de dados enviados pela Internet, evitando possíveis monitoramentos e/ou alterações dos dados. Este protocolo é amplamente utilizado em aplicativos como mensagens, e-mail, etc.

Reviewer: ```suggestion
Transport Layer Security (TLS) é um protocolo projetado para fornecer maior segurança à comunicação através de uma rede. Garante a entrega segura de dados enviados pela Internet, evitando possíveis monitoramentos e/ou alterações dos dados. Este protocolo é amplamente utilizado em aplicações como mensagens, e-mail, etc.

```

## Problema relacionado 

> Sem o TLS, informações confidenciais, como hábitos de navegação, correspondência por e-mail, bate-papos on-line e chamadas de conferência, podem ser facilmente rastreadas e modificadas por outras pessoas durante a transmissão. Permitir que os aplicativos de servidor e cliente suportem TLS garante que os dados transmitidos entre eles sejam criptografados e não possam ser visualizados por terceiros.

Reviewer: ```suggestion
Sem o TLS, informações confidenciais, como hábitos de navegação, correspondência por e-mail, bate-papos on-line e chamadas de conferência, podem ser facilmente rastreadas e modificadas por outras pessoas durante a transmissão. Permitir que as aplicações de servidor e cliente suportem TLS garante que os dados transmitidos entre elas sejam criptografados e não possam ser visualizados por terceiros.

```

## Como isso ajuda

> O TLS usa uma combinação de técnicas de codificação que fornecem segurança ao transmitir dados através de uma rede. O TLS permite uma conexão criptografada entre um aplicativo cliente e um servidor, tal como um navegador da web e um site bancário. Ele também permite que os aplicativos clientes identifiquem positivamente o servidor que está conectado, o que reduz o risco de um cliente falar com um site fraudulento. Isso garante que terceiros não possam ver e monitorar dados transmitidos entre aplicativos usando o TLS, o que protege informações confidenciais e privadas, como números de cartão de crédito, senhas, localização, etc.

Reviewer: ```suggestion
O TLS usa uma combinação de técnicas de codificação que fornecem segurança ao transmitir dados através de uma rede. O TLS permite uma conexão criptografada entre uma aplicação cliente e um servidor, tal como um navegador da web e um site bancário. Ele também permite que as aplicações clientes identifiquem positivamente o servidor que está conectado, o que reduz o risco de um cliente se comunicar com um site fraudulento. Isso garante que terceiros não possam ver e monitorar dados transmitidos entre aplicações usando o TLS, o que protege informações confidenciais e privadas, como números de cartão de crédito, senhas, localização, etc.

```
